sonata.core

```diff
@@ -16,6 +16,7 @@ filesets:
     files:
       - rtl/fpga/top_sonata.sv
       - rtl/fpga/clkgen_sonata.sv
+      - rtl/fpga/rst_ctrl.sv
```

Comment:
Is this chappy checked in?

Reply:
Thanks for noticing. I've just pushed it to the branch.